app/assets/javascripts/expand.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function() {
+  $(document).on('click', '.timeline-event', function(event){
+
+    var request = $.ajax({
+      method: "GET",
+      url: "/api/v1/events/" + $(this).attr('id'),
+    });
+
+    var myNode = document.getElementById("expand-event");
+    while (myNode.firstChild) {
+      myNode.removeChild(myNode.firstChild);
+    }
+
+
+
+    request.success(function(response) {
+      var url = document.URL.split("/");
+      var timeline_id = url[url.length - 1];
+      var id = response.event.id;
+      var title = response.event.title;
+      var description = response.event.description;
+      var title_header = document.createElement("h1");
+      var para = document.createElement("p");
+      var title_node = document.createTextNode(title);
+      var desc_node = document.createTextNode(description);
+
+
+      title_header.appendChild(title_node);
+      para.appendChild(desc_node);
+
+      var element = document.getElementById("expand-event");
+
+
+      //Create an input type dynamically.
+      var update_button = document.createElement("a");
+      var delete_button = document.createElement("a");
+
+      var update_text = document.createTextNode("update event");
+      var delete_text = document.createTextNode("delete event");
```

Comment:
Identifier 'delete_text' is not in camel case.<br>Mixed double and single quotes.